tools/renew-certificates.sh

```diff
@@ -10,13 +11,31 @@ for file in $(find "${MOZIOT_HOME}/etc/renewal" -type f -name '*.conf'); do
     sed -i -e 's/^must_staple.*/must_staple = False/' "${file}"
 done
 
+for file in $(find "${MOZIOT_HOME}/etc/accounts" -type f -name regr.json); do
+    if ! grep -q new_authzr_uri "${file}"; then
+        python - <<EOF
+import json
+
+with open('${file}') as f:
+    data = json.load(f)
+
+data['new_authzr_uri'] = 'https://acme-v01.api.letsencrypt.org/acme/new-authz'
+
+with open('${file}', 'wt') as f:
+    json.dump(data, f)
+EOF
+    fi
+done
+
 # NOTE: --renew-hook is deprecated and has been replaced by --deploy-hook.
 #       However, the version of certbot in raspbian stretch doesn't have
 #       --deploy-hook support, so we're sticking with the old option for now.
 certbot renew \
     --config-dir "${MOZIOT_HOME}/etc" \
     --logs-dir "${MOZIOT_HOME}/var/log" \
     --work-dir "${MOZIOT_HOME}/var/lib" \
-    --renew-hook "/home/pi/mozilla-iot/gateway/tools/deploy-certificates.sh"
+    --renew-hook "${_dir}/deploy-certificates.sh"
 
-chown -R pi:pi "${MOZIOT_HOME}/etc" "${MOZIOT_HOME}/var"
+# chown the certs as uid/gid 1000, which maps to 'pi' in the Raspberry Pi image,
+# and 'node' in the Docker image.
```

Comment:
It would be good to make sure that the uid 1000 exists and is pi/node (i.e. be nice to people if this is running on their laptop/generic linux box)

Reply:
Although the use of MOZIOT_HOME makes it hard to justify adding that logic. I'm okay with or without a check